tests/api/archive_basic_test.rs

```diff
@@ -189,3 +194,300 @@ async fn get_law_pdf_request_with_incorrect_path_expect_not_found() {
     let expected = true;
     assert_eq!(actual, expected);
 }
+
+#[actix_web::test]
+async fn test_resolve_root_stele_all_repositories_request_with_full_path_expect_success() {
+    let archive_path =
```

Comment:
Although these tests initialize the basic archive, they also now test the `_stelae` endpoint, while other tests test dynamic routes, which seems confusing.

suggestion: move new tests to a new test file

Also, please see our [TESTING.md](https://github.com/openlawlibrary/stelae/blob/main/TESTING.md#testing-conventions) style guide for adding new tests. Usually we break down asserts into multiple tests. Ideally a single assert per test, but more realistically a single data repo per test in this case. But I wouldn't worry about this too much right now.